0016-pre-shared-key/0016-pre-shared-key.md

```diff
@@ -0,0 +1,57 @@
+# Pre Shared Key
+
```

Comment:
It would be useful to put it in context of an ILP transfer. Where does it fit into the orchestration and where do these messages get sent and between whom.